sylius/1/versions

```diff
@@ -1 +1,2 @@
 1.4.1|Sylius-1.4.1.tar.gz
```

Comment:
we don't generally delete the old versions of applications. since this PR deletes 1.4.1 we should either reverse that and NOT delete 1.4.1 or we need to remove this line from the versions list

Reply:
yea, the tar files were removed by mistake, should be ok now.